sympy/vector/operators.py

```diff
@@ -240,6 +240,8 @@ def divergence(vect, coord_sys=None, doit=True):
     if len(coord_sys) == 0:
         return S.Zero
     elif len(coord_sys) == 1:
+        if isinstance(vect, (Cross, Curl, Gradient)):
+            return Divergence(vect)
```

Comment:
Such expression  `divergence(Cross(a.x*a.i, a.y*a.j))` goes to the `elif len(coord_sys) == 1` condition so we can't do the calculation which are below